ledger-tool/src/main.rs

```diff
@@ -654,6 +655,58 @@ fn load_bank_forks(
     )
 }
 
+fn print_account(pubkey: &Pubkey, account: &Account) {
+    let data_len = account.data.len();
+    println!("{}:", pubkey);
+    println!("  - balance: {} SOL", lamports_to_sol(account.lamports));
+    println!("  - owner: '{}'", account.owner);
+    println!("  - executable: {}", account.executable);
+    println!("  - rent_epoch: {}", account.rent_epoch);
+    println!("  - data: '{}'", base64::encode(&account.data));
```

Comment:
@mvines Also, I've started to encoding from base58 to base64 because I've experienced so slow encoding for large data (maybe quadric time; O(n^2))..

It seems that @nearprotocol people are also experiencing this: https://github.com/nearprotocol/nearcore/blob/master/Cargo.toml#L84

Reply:
How much slower is bs58 in this case?   Also do the opt-level  = 3 do anything?   It's going to get really confusing for the user if we mix base58 and base64, especially if there's no prefix.   

How about we use the same output as `solana account`?    That output could be factored out of `solana account` and put in a common function, like https://github.com/solana-labs/solana/blob/9bba27a3aaab033fed57ccb8b02e2d908c9b3aaf/cli/src/display.rs#L67

Reply:
@mvines Ok, I'll do the `solana account` path. :) Hopefull, I'll also add `base64` to the shiny `--output`.

> Also do the opt-level = 3 do anything?

sadly, not much... ;)

> How much slower is bs58 in this case?


For the record: well, it seems that it's well-known that base58 is slow algorithmically by design. its O(n) is quadratic. 

- https://github.com/aeternity/aeternity/pull/1733: <i>The base58 encoding/decoding is hopelessly slow for large byte arrays.</i>
- https://news.ycombinator.com/item?id=18408008: <i>but is still quadratically expensive and should be used for a reasonably short, known-length data like hashes.</i>
- https://github.com/tuupola/base58: <i>That said, Base58 is well suited for decoding big integers but is not designed to decode long portions of binary data.</i>


My quick observation also aligns with these quick findings:

```
$ cat runtime/src/bank.rs | env time -v base58 > /dev/null
        Command being timed: "base58"
        User time (seconds): 142.34
        System time (seconds): 1.39
        Percent of CPU this job got: 100%
        Elapsed (wall clock) time (h:mm:ss or m:ss): 2:23.73
        Average shared text size (kbytes): 0
        Average unshared data size (kbytes): 0
        Average stack size (kbytes): 0
        Average total size (kbytes): 0
        Maximum resident set size (kbytes): 20044
        Average resident set size (kbytes): 0
        Major (requiring I/O) page faults: 0
        Minor (reclaiming a frame) page faults: 1532691
        Voluntary context switches: 3
        Involuntary context switches: 162
        Swaps: 0
        File system inputs: 0
        File system outputs: 0
        Socket messages sent: 0
        Socket messages received: 0
        Signals delivered: 0
        Page size (bytes): 4096
        Exit status: 0
==> 2020-04-28 16:43:20.819247997|0
$ cat runtime/src/bank.rs runtime/src/bank.rs | env time -v base58 > /dev/null
        Command being timed: "base58"
        User time (seconds): 485.85
        System time (seconds): 6.86
        Percent of CPU this job got: 100%
        Elapsed (wall clock) time (h:mm:ss or m:ss): 8:12.70
        Average shared text size (kbytes): 0
        Average unshared data size (kbytes): 0
        Average stack size (kbytes): 0
        Average total size (kbytes): 0
        Maximum resident set size (kbytes): 21920
        Average resident set size (kbytes): 0
        Major (requiring I/O) page faults: 0
        Minor (reclaiming a frame) page faults: 6581747
        Voluntary context switches: 5
        Involuntary context switches: 317
        Swaps: 0
        File system inputs: 0
        File system outputs: 0
        Socket messages sent: 0
        Socket messages received: 0
        Signals delivered: 0
        Page size (bytes): 4096
        Exit status: 0
```